### ISSUE TITLE
Bacterial encephalitis

### ISSUE THREAD

**Issue by oleraj** [obophenotype/human-phenotype-ontology#7442]
**Preferred term label:**
Bacterial encephalitis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Encephalitis caused by bacterial infection (?).
A few papers mentioning this term:
- PMID: [31525232](https://www.ncbi.nlm.nih.gov/pubmed/31525232)
- https://www.thelancet.com/journals/lancet/article/PIIS1473-3099(10)70222-X/fulltext

Definition used for encephalitis in the second paper above:
"The case definition included any person of any age admitted to hospital with encephalopathy (altered consciousness that persisted for longer than 24 h, including lethargy, irritability, or a change in personality and behaviour) and with two or more of the following: fever or history of fever (≥38°C) during the presenting illness; seizures and/or focal neurological findings (with evidence of brain parenchyma involvement); CSF pleocytosis (more than four white blood cells per μL); electroencephalographic (EEG) findings indicative of encephalitis; and abnormal results of neuroimaging (CT or MRI) suggestive of encephalitis."

**Parent term (use hpo.jax.org/app)**
Infectious encephalitis HP:0002383

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Manuscript in preparation

**Your nano-attribution (ORCID)**
0000-0002-6310-0434

**Comment by pnrobinson** [comment]
added as HP:0034387, thanks @oleraj
